vllm/config.py

```diff
@@ -2063,6 +2063,17 @@ def __post_init__(self) -> None:
                 raise ValueError(
                     "num_redundant_experts must be non-negative, but got "
                     f"{self.num_redundant_experts}.")
+            if not self.enable_expert_parallel:
+                raise ValueError(
+                    "enable_expert_parallel must be True to use EPLB.")
+            if self.tensor_parallel_size <= 1:
+                raise ValueError(
+                    "EPLB requires tensor_parallel_size to be greater than 1, "
+                    f"but got {self.tensor_parallel_size}.")
+            if self.data_parallel_size <= 1:
+                raise ValueError(
+                    "EPLB requires data_parallel_size to be greater than 1, "
+                    f"but got {self.data_parallel_size}.")
```

Comment:
This will only allow expert parallel if both TP and DP are enabled, the condition should be if either of them are enabled

```suggestion
            if self.tensor_parallel_size <= 1:
                raise ValueError(
                    "EPLB requires tensor_parallel_size to be greater than 1, "
                    f"but got {self.tensor_parallel_size}.")
            if self.data_parallel_size <= 1:
                raise ValueError(
                    "EPLB requires data_parallel_size to be greater than 1, "
                    f"but got {self.data_parallel_size}.")
```